regional_tiered_cache.go

```diff
@@ -0,0 +1,82 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// RegionalTieredCache is the structure of the API object for the regional tiered cache
+// setting.
+type RegionalTieredCache struct {
+	ID         string    `json:"id,omitempty"`
+	ModifiedOn time.Time `json:"modified_on,omitempty"`
+	Value      string    `json:"value"`
+}
+
+// RegionalTieredCacheDetailsResponse is the API response for the regional tiered cache
+// setting.
+type RegionalTieredCacheDetailsResponse struct {
+	Result RegionalTieredCache `json:"result"`
+	Response
+}
+
+type zoneRegionalTieredCacheSingleResponse struct {
+	Response
+	Result RegionalTieredCache `json:"result"`
+}
```

Comment:
Unsure if this separate struct is preferred (since the contents are the same)... saw this pattern introduced here: https://github.com/cloudflare/cloudflare-go/pull/1278/commits/f047e5560a5c76484ea329b8ee25f2d477f2d7ec#diff-46ac00cf63f77e30464ff4ec022cc846b403a56c9e91b9b84f1de982aae6419cR30-R33

Reply:
yep, this seems 👌